lib/storage/src/content_manager/toc.rs

```diff
@@ -125,6 +129,8 @@ impl TableOfContent {
             raft_state: Arc::new(std::sync::Mutex::new(RaftState::default())),
             #[cfg(feature = "consensus")]
             propose_sender: None,
+            #[cfg(feature = "consensus")]
+            on_meta_op_apply: std::sync::Mutex::new(HashMap::new()),
```

Comment:
Are you using `std::sync::Mutex` instead of `tokio::sync::Mutex` on purpose, or it is just simpler for now?

Reply:
Generally tried to keep consensus async-less for steady performance and simplicity.

Reply:
Since the consensus uses its own thread, I guess that is Ok.